CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Current Master
 
 - Fix bug where account reset did not work with custom RPC providers.
+- Fix for Brave i18n getAcceptLanguages [#4270](https://github.com/MetaMask/metamask-extension/issues/4270)
```

Comment:
@jennypollack we add changelog entries under "current master" and then move that to a version number of release